SwiftGen/main.swift

```diff
@@ -45,6 +46,6 @@ let main = Group {
   $0.addCommand("fonts", "generate code for your UIFonts", fontsCommand)
 }
 
-let version = Bundle(for: GenumKit.GenumTemplate.self)
+let version = Bundle.main
   .infoDictionary?["CFBundleShortVersionString"] as? String ?? "0.0"
 main.run("SwiftGen v\(version)")
```

Comment:
Could be interesting to fetch versions for SwiftGenKit and StencilSwiftKit too so we list them all there?